src/enum_tags.h

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
Please add this file to [Makefile.am](https://github.com/EasyRPG/liblcf/blob/master/Makefile.am#L115). This is especially needed for distribution archives.